src/python/pants/backend/jvm/subsystems/zinc.py

```diff
@@ -372,7 +372,7 @@ def _compile_compiler_bridge(self, context):
     if not os.path.exists(bridge_jar):
       res = self._run_bootstrapper(bridge_jar, context)
       context._scheduler.materialize_directories((
-        DirectoryToMaterialize(get_buildroot(), res.output_directory_digest),
+        DirectoryToMaterialize("", res.output_directory_digest),
```

Comment:
The vast majority of `path`s end up now being `""`, which means to materialize the digest directly into the build root with no prefixed path. So, in a followup PR, I plan to set the default value of `path` to `""`.

(Note we use `""` instead of `./` to avoid the Rust path becoming `/path/to/pants/./foo.txt`, which technically works, but I don't think is what we intend for.)